experimental/PropertyDDS/examples/property-inspector/src/theme.ts

```diff
@@ -7,7 +7,7 @@ import "@hig/fonts/build/ArtifaktElement.css";
 import { createTheme } from "@material-ui/core/styles";
 import { ToggleButtonClassKey } from "@material-ui/lab/ToggleButton";
 
-declare module "@material-ui/core/styles/overrides" {
+declare module "@material-ui/core/styles" {
```

Comment:
Not quite sure what the changes in this file are doing. But if they're necessary, shouldn't they also be necessary in `experimental/PropertyDDS/packages/property-inspector-table/src/Theme.tsx`?

Reply:
Reverting these changes yields:

```
@fluid-experimental/property-inspector: error during command 'tsc'
@fluid-experimental/property-inspector: src/theme.ts:10:16 - error TS2664: Invalid module name in augmentation, module '@material-ui/core/styles/overrides' cannot be found.
@fluid-experimental/property-inspector: 
@fluid-experimental/property-inspector: 10 declare module "@material-ui/core/styles/overrides" {
@fluid-experimental/property-inspector:                   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
@fluid-experimental/property-inspector: 
@fluid-experimental/property-inspector: src/theme.ts:24:5 - error TS2322: Type '{ MuiButton: { root: { borderRadius: string; }; }; MuiToggleButton: { root: { "&:not(:first-child)": { borderLeft: string; marginLeft: string; }; }; }; }' is not assignable to type 'Overrides'.
@fluid-experimental/property-inspector:   Object literal may only specify known properties, but 'MuiToggleButton' does not exist in type 'Overrides'. Did you mean to write 'MuiStepButton'?
@fluid-experimental/property-inspector: 
@fluid-experimental/property-inspector:  24     MuiToggleButton: {
@fluid-experimental/property-inspector:         ~~~~~~~~~~~~~~~~~~
@fluid-experimental/property-inspector:  25       root: {
@fluid-experimental/property-inspector:     ~~~~~~~~~~~~~
@fluid-experimental/property-inspector: ... 
@fluid-experimental/property-inspector:  30       },
@fluid-experimental/property-inspector:     ~~~~~~~~
@fluid-experimental/property-inspector:  31     },
@fluid-experimental/property-inspector:     ~~~~~
@fluid-experimental/property-inspector: 
@fluid-experimental/property-inspector: 
@fluid-experimental/property-inspector: Found 2 errors.
```